### ISSUE TITLE
LOW PRIORITY SINGLE DRAFT IS RETARDED

### ISSUE THREAD

**Issue by patrickdx** [ValveSoftware/Dota2-Gameplay#19988]
### Description

what kind of retard at valve thought it was a good idea to force low prio players to queue single draft?

its 10+ minute queueing the most common servers na east/west euwest. and to top if off u have to win?!?!?!?! it dosen't matter u want us to play for win when its a 50% chance there is a afk on each team just a coin flip really + ruining games for single draft only players because now its full of only low prio players b/c of ur horrible decision that u think is right )

**Comment by evil-inside1332** [comment]
the low priority pool is separate from the regular player pool
at least that's what we've been told

**Comment by Jordan11599** [comment]
This is correct or feels to be correct. It's usually almost always correct in punishing those who should be punished (due to toxicity) but towing the line is frustrating when in some cases players report out of spite.
